v2/yarpcgrpc/peer.go

```diff
@@ -44,44 +44,30 @@ type grpcPeer struct {
 	stoppedErr error
 }
 
-func (t *Transport) newPeer(address string, options *dialOptions) (*grpcPeer, error) {
-	dialOptions := append([]grpc.DialOption{
-		grpc.WithUserAgent(UserAgent),
-		grpc.WithDefaultCallOptions(
-			grpc.CallCustomCodec(customCodec{}),
-			grpc.MaxCallRecvMsgSize(t.options.clientMaxRecvMsgSize),
-			grpc.MaxCallSendMsgSize(t.options.clientMaxSendMsgSize),
-		),
-	}, options.grpcOptions()...)
-
-	clientConn, err := grpc.Dial(address, dialOptions...)
+func (d *dialerInternals) newPeer(id yarpc.Identifier) (*grpcPeer, error) {
+	clientConn, err := grpc.Dial(id.Identifier(), d.dialOptions...)
```

Comment:
Ah, now I see where the `dialOptions` are applied. Feel free to ignore my comment from earlier.